### PR TITLE
[urplay] Extract timestamp

### PR DESCRIPTION
**Pull request by 0x9fff00**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Extracts timestamp from `uploadDate` meta tag.

### CODE REVIEW

**Comment by dstftw** [comment]
Does not work:
```
> py -2.7 .\youtube_dl\__main__.py https://urskola.se/Produkter/190031-Tripp-Trapp-Trad-Sovkudde -F
[URPlay] 190031: Downloading webpage
[URPlay] 190031: Downloading JSON metadata
[URPlay] 190031: Downloading m3u8 information
[URPlay] 190031: Downloading f4m manifest
[URPlay] 190031: Downloading MPD manifest
[URPlay] 190031: Downloading m3u8 information
[URPlay] 190031: Downloading f4m manifest
[URPlay] 190031: Downloading MPD manifest
WARNING: unable to extract uploadDate; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

File: youtube_dl/extractor/urplay.py [hunk 1]
```diff
@@ -51,6 +52,7 @@ def _real_extract(self, url):
             'title': urplayer_data['title'],
             'description': self._og_search_description(webpage),
             'thumbnail': urplayer_data.get('image'),
+            'timestamp': parse_iso8601(self._html_search_meta('uploadDate', webpage)),
```
Comment (dstftw) [inline youtube_dl/extractor/urplay.py]: `unified_timestamp`.

**Comment by 0x9fff00** [comment]
Done.

**Comment by 0x9fff00** [comment]
Fixed for UR Skola.

File: youtube_dl/extractor/urplay.py [hunk 2]
```diff
@@ -2,18 +2,31 @@
...
 from .common import InfoExtractor
+from ..utils import unified_timestamp
 
 
 class URPlayIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?ur(?:play|skola)\.se/(?:program|Produkter)/(?P<id>[0-9]+)'
     _TESTS = [{
-        'url': 'http://urplay.se/program/190031-tripp-trapp-trad-sovkudde',
-        'md5': 'ad5f0de86f16ca4c8062cd103959a9eb',
+        'url': 'https://urplay.se/program/209701-framtidens-dromhem-min-oppna-fristad',
```
Comment (dstftw) [inline youtube_dl/extractor/urplay.py]: Geo restricted. Test will always fail.

Comment (0x9fff00) [reply youtube_dl/extractor/urplay.py]: Replaced with unrestricted video.

**Pull request merged**